package.json

```diff
@@ -41,6 +41,9 @@
   "browserslist": [
     "> 1%, not dead, not ie 11, not op_mini all"
   ],
+  "engines": {
+    "node": ">=16.19.0"
+  },
```

Comment:
Can we not put this on the published package.json? This affects those who install three from NPM.

Reply:
That is not the case.  Test it out.

Reply:
Have you heard of `--ignore-engines` or have configured a Node package before? This is very much the case and I'm happy to contribute an example or integration tests if that's required to substitute caution before shipping breaking user-facing changes.

Reply:
You can easily test in under a minute and see that is not an issue.  We don't install tooling or register executable commands so we don't have that problem.